src/Spec2-Core/SpPresenter.class.st

```diff
@@ -1098,6 +1137,14 @@ SpPresenter >> windowIcon [
 	^ windowIcon
 ]
 
+{ #category : 'private' }
+SpPresenter >> withPopoverDo: aBlock [
+	"Execute a block in the first popover in the DOM"
+	
+	self isPopover ifTrue: [ ^ aBlock cull: self ].
+	self owner ifNotNil: [ 
+		self owner withPopoverDo: aBlock ]
+
```

Comment:
@Ducasse there is a missing method closing `]`.
that's why Tonel cannot parse this class.

Reply:
tx!